quixstreams/platforms/quix/config.py

```diff
@@ -185,6 +147,26 @@ def workspace_meta(self) -> dict:
             self.get_workspace_info()
         return self._workspace_meta
 
+    @property
+    def librdkafka_connection_config(self) -> ConnectionConfig:
+        if not self._librdkafka_connection_config:
+            self._get_librdkafka_connection_config()
+        return self._librdkafka_connection_config
```

Comment:
What if we simply get the librdkafka on `__init__`?
This way we wouldn't need to update the `self._librdkafka_connection_config` attribute.

Reply:
I def agree with the idea in general since we probably are using this class for that purpose 99% of the time.

Unfortunately it's a bit linked to the `get_workspace_id`, which itself could throw an exception if the token was a PAT and no Workspace ID was defined (and thus has multiple possible workspace IDs...this situation requires users to pass a topic to `get_workspace_id` manually to search for the proper workspace ID). Not that I think anyone is doing anything that way right now, but it is an issue regardless.

If we set it up in `__init__`, it wont allow creating an instance at all in that circumstance due to said exception.

TL;DR This should probably be part of a separate story to clean up this class (and its workspace management in general), as we already have noted before.

Thoughts?